frontend/packages/dev-console/integration-tests/tests/serverless.scenario.ts

```diff
@@ -0,0 +1,134 @@
+    await catalogView.categoryTabs.get(0).click();
+    await catalogPageView.catalogTileFor('OpenShift Serverless Operator').click();
+    await browser.wait(until.visibilityOf(installServerlessOperator));
+    await installServerlessOperator.element(by.linkText('Install')).click();
+    await operatorHubView.createSubscriptionFormLoaded();
+    expect(operatorHubView.createSubscriptionFormName.getText()).toEqual(
+      'OpenShift Serverless Operator',
+    );
+    await browser.wait(until.visibilityOf(operatorHubView.createSubscriptionFormInstallMode));
+    await operatorHubView.allNamespacesInstallMode.click();
+
+    expect(operatorHubView.createSubscriptionError.isPresent()).toBe(false);
+    expect(operatorHubView.createSubscriptionFormBtn.getAttribute('disabled')).toEqual(null);
+    await operatorHubView.createSubscriptionFormBtn.click();
+    await crudView.isLoaded();
+    await browser.get(`${appHost}/operatorhub/ns/${testName}`);
+    await crudView.isLoaded();
+    await catalogPageView.clickFilterCheckbox('installState-installed');
+    // Extend the default jasmine timeout interval just in case it takes a while for the htpasswd idp to be ready
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = JASMINE_EXTENDED_TIMEOUT_INTERVAL;
+    expect(catalogPageView.catalogTileFor('OpenShift Serverless Operator').isDisplayed()).toBe(
+      true,
+    );
+  });
+
+  it(`displays Operator in "Cluster Service Versions" view for "${testName}" namespace`, async () => {
+    await retry(() => catalogPageView.catalogTileFor('OpenShift Serverless Operator').click());
+    await browser
+      .wait(until.presenceOf(installServerlessOperator), 1000)
+      .then(() => browser.sleep(500));
+    await operatorHubView.viewInstalledOperator();
+    await crudView.isLoaded();
+    await browser.wait(
+      until.visibilityOf(operatorView.rowForOperator('OpenShift Serverless Operator')),
+    );
+    // Extend the default jasmine timeout interval just in case it takes a while for the htpasswd idp to be ready
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = JASMINE_EXTENDED_TIMEOUT_INTERVAL;
+    expect(operatorView.rowForOperator('OpenShift Serverless Operator').isDisplayed()).toBe(true);
+  });
+
+  it(`create knative-serving namespace and apply serving.yaml`, async () => {
+    await execSync(`oc new-project knative-serving`);
+    await execSync(
+      `oc apply -f ./packages/dev-console/integration-tests/views/serverless-service.yaml`,
+    );
+    expect(true).toBeTruthy();
+  });
+
+  it('Serverless is installed', async () => {
+    hardRefreshPage();
+    await crudView.isLoaded();
+    await browser.wait(until.presenceOf(sidenavView.navSectionFor('Serverless')));
+    // Extend the default jasmine timeout interval just in case it takes a while for the htpasswd idp to be ready
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = JASMINE_EXTENDED_TIMEOUT_INTERVAL;
+    hardRefreshPage();
+    await sidenavView.clickNavLink(['Serverless', 'Services']);
```

Comment:
Expect for `serverless` to be present in sidebar and then expect for children for it to be `Services /Revisions/Revisions `

then on click maybe you can check for something in the view being open on click. WDYT?

Reply:
It wasn't the part of this automation. That's why I haven't added that. Also this must be taken care from the console team. The purpose of this is just the validation. So what you are saying can be done but then it must be included in the console automation and not ours.

Reply:
am not sure why is that, `Serverless` you are checking on console only and even that's the case for serverless we should assert if serverless is there.